src/ex1/cli.js

```diff
@@ -0,0 +1,50 @@
+import { program } from 'commander';
+import color from'colors-cli';
+import ItemManager from './ItemManager.js';
+const itemManager = new ItemManager();
+
+
+
+program
+    .name('To Do List')
+    .description('This is my TODO List')
+    .version("1.0.0");
+program
+    .command('add')
+    .description('Add a new todo')
+    .argument("<text>", "Task")
+    .action(async (first) => {
+        await itemManager.addTask(first);
+        console.log(color.yellow("New todo was added successfully"));
```

Comment:
in this case you can use try catch and console in case of an error , this will be more friendly to the user